octane-visual-studio-plugin-tests/packages.config

```diff
@@ -1,6 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="MicroFocus.Adm.Octane.Api.Core" version="16.0.200.1" targetFramework="net45" />
-  <package id="MSTest.TestAdapter" version="1.2.0" targetFramework="net45" />
-  <package id="MSTest.TestFramework" version="1.2.0" targetFramework="net45" />
-</packages>
```

Comment:
what was this config used for?

Reply:
I've upgraded the project to use package reference. These packages were moved in the .csproj file with <PackageReference> tags